msu/hooks/items/item_container.nut

```diff
@@ -9,6 +10,8 @@
 
 	o.getActionCost = function( _items )
 	{
+		if (this.m.IsMSUHandledItemAction) return 0;
```

Comment:
Doesn't this mess with tooltips?

Reply:
No, because these are only for AI. Also, returning 0 for a player doesn't mess any tooltip, it just mentions the cost of the item swap to be 0 AP.

Reply:
What do you mean? That function is used in tooltips. And showing 0 cost when it's not 0 cost is bad?

Reply:
He's saying that IsMSUHandledItemAction would only be true for AI skills @TaroEld (that's the first part anyway)

Reply:
I see